java/tools/src/main/java/org/apache/tsfile/tools/TsFileTool.java

```diff
@@ -158,20 +167,25 @@ private static boolean writeTsFile(String fileName, List<String> lineList) {
         return false;
       }
     } catch (Exception e) {
-      e.printStackTrace();
-      LOGGER.error("Failed to write file: " + tsFile);
+      LOGGER.error("Failed to write file" + tsFile.getAbsolutePath() + ":{} ", e.getMessage());
       return false;
     } finally {
       if (writer != null) {
         try {
           writer.close();
         } catch (IOException e) {
-          e.printStackTrace();
+          LOGGER.error("Failed to close file" + tsFile.getAbsolutePath() + ":{}", e.getMessage());
```

Comment:
Are you sure the exception stack is unnecessary? If not, may use
 LOGGER.error("Failed to close file: " + tsFile.getAbsolutePath(), e);

Reply:
Please also check other places.